docs/live-data-VehicleLiveDataRetriever.js

```
@@ -0,0 +1,262 @@
+      setCatConverterTemp(`${value}${unit ? ` ${unit}` : ''}`),
+  };
+
+  static _instance = null;
+
+  constructor(ecuDataRetriever = null) {
+    if (VehicleLiveDataRetriever._instance) {
+      ecuDataRetriever &&
+        (VehicleLiveDataRetriever._instance.ecuDataRetriever =
+          ecuDataRetriever);
+
+      return VehicleLiveDataRetriever._instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)